test/boost/view_schema_test.cc

```diff
@@ -2993,6 +2993,16 @@ SEASTAR_TEST_CASE(test_base_column_in_view_pk_complex_timestamp) {
     });
 }
 
+namespace std {
+
+// Needed by `test_view_update_generating_writetime` below.
+ostream& operator<<(ostream& os, std::tuple<int, int, int> tuple) {
+    os << seastar::format("{}", tuple);
+    return os;
+}
+
+} // namespace std
+
```

Comment:
If we're improving readability, maybe we can also print what each value means (I see all tuples below have the same meaning, we could use a struct instead)

Reply:
That's a good idea, thanks!